src/Components/Poster.js

```diff
@@ -1,14 +1,17 @@
 import React from "react";
 import '../CSS_styling/Poster.css'
 
-const Poster = ({image, title, rating, releaseDate}) => {
+const Poster = ({image, title, rating, releaseDate, findDetails, id}) => {
   return (
     <div className="poster">
-      <img src={image} />
+      <img className='poster-img' src={image} alt={title} />
```

Comment:
Since you are visually displaying the title right below the image, you may not need alt text! Remember its redundant for the screen reader to say the same thing twice. If you decide to do that, make sure you have an empty alt tag so its not flagged alt=""

I was curious how Rotten Tomatoes does their alt text, and they got an 83 on Lighthouse because they don't have any alt attributes at all :(